packages/console/src/pages/Applications/components/CreateForm/index.tsx

```diff
@@ -61,6 +66,9 @@ const CreateForm = ({ onClose }: Props) => {
               </Radio>
             ))}
           </RadioGroup>
+          {errors.type?.type === 'required' && (
```

Comment:
`errors.type?.type` ?

Reply:
yes, the first type is a field name and the second one is error type, see https://react-hook-form.com/get-started#Handleerrors